### DISCUSSION TITLE
Multiple language support for pages (content)

### DISCUSSION THREAD

**Discussion by llann** [docmost/docmost#599] [Ideas]
Hello, would it be possible to offer the same page (content) in several languages? This would make it possible to manage business or help cases for external customers who don't all speak the same language, or for international companies to use the tool too.

Thanks!

**Comment by nums** [comment]
very interested in this feature too

**Comment by SamBouwer** [comment]
I'm using docmost to publish newsletters (I know there are other and maybe even better ways to accomplish that) and localization of content would be amazing!

**Comment by Uproot5456** [comment]
Using AI for generating a translated version of any given page could be useful here.

**Comment by wpskis** [comment]
Mintlify has a AI function build into to automatically translate - while burning a lot of tokens :D 

Would however just be great to be able to manage multiple language content natively within docmost instead of making two sapces or similar for separations.
